test/jpypetest/test_startup.py

```diff
@@ -16,24 +16,12 @@
 #
 # *****************************************************************************
 import jpype
-import common
 import subrun
 import functools
 import os
 from pathlib import Path
-import sys
 import unittest
 
-
-@functools.wraps(jpype.startJVM)
```

Comment:
why do you removed this pattern? We can only start the JVM once per thread.

Reply:
As for the reason for doing it - the ``functools.wraps`` didn't expose the right kind of callable to ``mypy``. I don't understand why not, but the pattern I found on the mypy issue tracker was to create a decorator along the lines of https://github.com/python/mypy/issues/1927#issuecomment-829988588.

In the end, the test pattern being used here wasn't ideal anyway - it wasn't obvious where the ``assert`` was happening (inside the run call), so splitting those things does add a bit of duplication, but makes it much clearer (IMO) what is going on, and it meant that I didn't have to deal with ``mypy`` issues :wink:

Reply:
Unfortunately, subrun will always be very slow as it has to produce a full new Python process and launch a JVM in it.  But that is the only way test it due to the one JVM per process restriction.